cassandra/metadata.py

```diff
@@ -168,10 +168,14 @@ def _rebuild_all(self, parser):
         current_keyspaces = set()
         for keyspace_meta in parser.get_all_keyspaces():
             current_keyspaces.add(keyspace_meta.name)
-            old_keyspace_meta = self.keyspaces.get(keyspace_meta.name, None)
+            old_keyspace_meta: KeyspaceMetadata = self.keyspaces.get(keyspace_meta.name, None)
             self.keyspaces[keyspace_meta.name] = keyspace_meta
             if old_keyspace_meta:
```

Comment:
🔧 This type hint is obviously incorrect - it should be `KeyspaceMetadata | None`.

Reply:
`|` is not supported by python 3.8

Reply:
typing.Optional is

Reply:
yes, replaced with `Optional`